pkg/operators/ebpf/params.go

```diff
@@ -102,7 +102,7 @@ func (i *ebpfInstance) populateParam(t btf.Type, varName string) error {
 	}
 
 	// Fill additional information from metadata
-	paramInfo := i.config.Sub("params." + varName)
+	paramInfo := i.config.Sub("params.ebpf." + varName)
```

Comment:
Hmm, I was gonna ask whether we should keep supporting the old format for a bit longer, but looking at the metadata, it seems we never used `params.` but `ebpfParams.` for the gadgets... 🙈

Reply:
Yeah, there were also some questions about the compatibility in #3490.